README.md

```diff
@@ -42,6 +42,9 @@ Yes! Astra Pro settings that are available with customizer can be imported/expor
 
 ## Changelog ##
 
+### 1.0.6 ###
+- Fix: PHP notice while importing the settings - Undefined index astra-addons
```

Comment:
```suggestion
- Fix: Compatibility with WordPress 6.0.
```

Reply:
Correct me if I wrong @nileshc-bsf, that was old issue as per as I remember. So PHP notice changelog seems to be more valid here. Right ??

Reply:
@imnavanath - Yes Correct no need to modify the changelog, your thoughts @premanshup

Reply:
Okay @imnavanath and @nileshc-bsf! We can go ahead with the original changelog, marking this as resolved.